lib/deploygate/command_builder.rb

```diff
@@ -86,12 +86,14 @@ def run
       command ADD_DEVICES do |c|
         c.syntax = 'dg add-devices'
         c.description = I18n.t('command_builder.add_devices.description')
+        c.option '--message STRING', String, I18n.t('command_builder.add_devices.message')
```

Comment:
It looks better for me to use `command_builder.deploy.message` to represent *reuse*. What do you think? @anoworl 

ref: https://github.com/DeployGate/deploygate-cli/pull/305/files#diff-79f3cff459436871e580bb0a25dc3de5b4c3830f2bb92915d7d811ad65574d85R94

Reply:
Thank you. True. I will change it.

Reply:
I did! https://github.com/DeployGate/deploygate-cli/pull/305/commits/21cd2798026465465a46f499064ec078b2553be2

And checked "dg add-devices -h" outputs.